common/components/meta-list/template.njk

```diff
@@ -11,6 +11,14 @@
         <dd class="app-meta-list__value">
           {{ item.value.html | safe if item.value.html else item.value.text }}
         </dd>
+
+        {% if item.action.html or item.action.text %}
+          <dd class="app-meta-list__action {% if item.action.classes -%} {{ item.action.classes }}{%- endif %}">
```

Comment:
Thanks for these changes. This feels much cleaner from a component perspective and removes the tie to the content of the element.

However, it would be good if this could be split into a separate commit with a little explanation why we changed it (maybe once everything has been rebased).

If you look back on that component with `git blame` embedded in the rest of this PR it will be a bit noisy.

Reply:
Rejigged